FrontEnd/src/components/adminPage/detail-view/DeleteModal.module.css

```diff
@@ -0,0 +1,48 @@
+.modal-window {
```

Comment:
I assume it's the modal cover, so the name is really confusing. It's better to call it `modal-cover` or `modal-backdrop`

Reply:
Done